src/patterns/dosdp-dev/abnormalConstrictionOfAnatomicalEntity.yaml

```
@@ -1,17 +1,24 @@
 pattern_name: abnormalConstrictionOfAnatomicalEntity
 pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormalConstrictionOfAnatomicalEntity.yaml
-description: "An anatomical entity that is being drawn together, compressed or squeezed physically, such as HP_0001650	Aortic valve stenosis."
+description: "An anatomical entity that is being drawn together, compressed or squeezed physically."
+
+# examples
+# http://purl.obolibrary.org/obo/HP_0001650      # Aortic valve stenosis
+# http://purl.obolibrary.org/obo/HP_0002043      # Esophageal stricture
+# http://purl.obolibrary.org/obo/MP_0003389      # constriction of pericardium
 
 contributors:
-- https://orcid.org/0000-0001-5208-3432
+- https://orcid.org/0000-0001-5208-3432  # Nicole Vasilevsky
+- https://orcid.org/0000-0001-6757-4744  # Arwa Ibrahim
+- https://orcid.org/0000-0001-8314-2140  # Ray Stefancsik
```

You need to fix the indentation of this line.
```suggestion
  - https://orcid.org/0000-0001-8314-2140  # Ray Stefancsik
```